unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs

```diff
@@ -278,13 +283,24 @@ private void MapRenderer_OnSceneInfoUpdated(MinimapMetadata.MinimapSceneInfo sce
             }
 
             centerTile /= (float)sceneInfo.parcels.Count;
+            float distance = float.PositiveInfinity;
+            Vector2 centerParcel = Vector2.zero;
+            foreach (var parcel in sceneInfo.parcels)
```

Comment:
you can extract this block into a method in `MinimapMetadata`: `GetNearestParcel(Vector2 from)`